src/features/views/l10n/messageIds.ts

```diff
@@ -2,46 +2,46 @@ import { ReactElement } from 'react';
 
 import { m, makeMessages } from 'core/i18n';
 
-export default makeMessages('feat.views', {
+export default makeMessages('feat.lists', {
```

Comment:
This should never have been renamed. It breaks all translations. Can someone please undo this, either @kaulfield23 or @ziggabyte and submit a new PR?

Reply:
I'll do it!